python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_resources.py

```diff
@@ -59,3 +61,10 @@ def test_basic_resource_request(
     client.start_resync(connector_id=connector_id, resync_parameters={"property1": ["string"]})
     assert len(all_api_mocks.calls) == 3
     assert f"{connector_id}/schemas/tables/resync" in all_api_mocks.calls[2].request.url
+
+    # poll calls
+    all_api_mocks.calls.reset()
```

Comment:
I know that this is just re-originating a method that exists prior to this PR, but I'm just looking at all the unexercised code paths [1] and [2]; would be nice to test failure conditions here if it's not hard. Won't block review on that, especially since the problem exists beyond just this PR

Reply:
That's totally fair - done in [a55e620](https://github.com/dagster-io/dagster/pull/26060/commits/a55e620a381353f1f69e9e32e3e6ffe62ae5ce70)